### PR TITLE
ACS-8542: add source to get agents request

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

File: hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/HxInsightClient.java [hunk 1]
```diff
@@ -82,8 +82,10 @@ public List<Agent> getAgents()
     {
         try
         {
+            String url = String.format("%s?source=%s", config.getAgentUrl(), applicationInfoProvider.getSourceId());
```
Comment (mpichura) [inline hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/HxInsightClient.java:85]: I mistyped the parameter name in the ticket.
```suggestion
            String url = String.format("%s?sourceId=%s", config.getAgentUrl(), applicationInfoProvider.getSourceId());
```

**Review by mpichura — APPROVED** [review]
LGTM. Parameter name must be changed (I mistyped that in the ticket).

File: e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/AgentsE2eTest.java [hunk 2]
```diff
@@ -97,6 +99,12 @@ void shouldReturnAgents()
                 .then().extract().response();
 
         // then
+
+        LoggedRequest loggedRequest = WireMock.findAll(WireMock.getRequestedFor(WireMock.anyUrl())).get(0);
+
+        assertNotNull(loggedRequest);
+        assertEquals("/agents?source=alfresco-dummy-source-id-0a63de491876", loggedRequest.getUrl());
```
Comment (azakrzewski-hy) [inline e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/AgentsE2eTest.java]: should we use the endpoint with "/integrations/agents?sourceId={sourceId}" instead "/agents?source={sourceId}" ?

Comment (azakrzewski-hy) [reply e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/AgentsE2eTest.java]: see that was mentioned above
LGTM

**Review by azakrzewski-hy — APPROVED** [review]

**Review by krdabrowski — APPROVED** [review]

File: hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/HxInsightClient.java (hunk 1 above)
Comment (Pawel-608) [reply hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/HxInsightClient.java]: fixed

**Pull request merged**